test/distributed/_composable/fsdp/test_fully_shard_training.py

```diff
@@ -1182,6 +1182,12 @@ def _test_2d_mlp_with_nd_mesh(
                 _optim.step()
             self.assertEqual(losses[0], losses[1])
 
+        for n, p in model.named_parameters():
+            self.assertIsInstance(p, DTensor)
+            self.assertEqual(p._spec.mesh.ndim, 2)
```

Comment:
nit: can we change this to `p.device_mesh.ndim` (public DTensor API) instead? thanks!